doc/rtd/tutorial/qemu-script.sh

```diff
@@ -22,6 +22,7 @@ EOF
 cat << EOF > meta-data
 instance-id: someid/somehostname
 local-hostname: jammy
+create_hostname_file: true
```

Comment:
Since this is a tutorial, I think we can remove this line (and probably the line above it) to avoid over-complicating things.

@holmanb , what do you think? Also...where is this file even used/referenced?

Reply:
> Since this is a tutorial, I think we can remove this line (and probably the line above it) to avoid over-complicating things.

+1 works for me

> Also...where is this file even used/referenced?

This file is referenced from the [Qemu tutorial debugging page](https://cloudinit.readthedocs.io/en/latest/tutorial/qemu-debugging.html).